.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
+end_of_line = lf
```

Comment:
This is better handled with a `.gitattributes` file, since that is enforced no matter what while this is only relevant for editors that have been modified to handle it.